README.npm.md

```diff
@@ -0,0 +1,209 @@
+
+```javascript
+import KedroViz from '@quantumblack/kedro-viz';
+import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
+
+const MyApp = () => <div style={{height: `100vh`}}><KedroViz data={json} /></div>; 
+```
+
+To use with NextJS:
+
+```javascript
+import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
+import dynamic from 'next/dynamic';
+
+const NoSSRKedro = dynamic(() => import('@quantumblack/kedro-viz'), {
+  ssr: false,
+});
+
+const MyApp = () => <NoSSRKedro data={json} />;
+```
+
+The JSON can be obtained by running:
+
+```bash
+kedro viz run --save-file=<path-to-save-kedro-viz-data-to-a-directory>
+```
+
+On successful execution of the command above, it will generate a folder at the specified directory, containing the following structure: 
+
+```
+<filename>/api/
+├── main
+├── nodes
+│   ├── 23c94afb
+│   ├── 28754fab
+│   ├── 2ab3579f
+│   ├── 329e963c
+│   ├── 369acf98
+│   └── ...
+└── pipelines
+    ├── __default__
+    ├── data_processing
+    ├── data_science
+    └── ...
+```
+
+Use the `main` file as the input JSON for the `data` prop in your Kedro-Viz component.
+
+We also recommend wrapping the `Kedro-Viz` component with a parent HTML/JSX element that has a specified height (as seen in the above example) in order for Kedro-Viz to be styled properly.
+
+**_Our documentation contains [additional examples on how to visualise with Kedro-Viz.](https://docs.kedro.org/en/stable/visualisation/index.html)_**
+
+## Props
+
+Below is the example with all possible props. 
+
+```
+<KedroViz
+    data={json}
+    options={
+      display: {
+        globalToolbar: true,
+        sidebar: true,
+        metadataPanel: true,
+        miniMap: true,
+        expandAllPipelines: false,
+      }
+      visible: {
+        exportBtn: true,
+        labelBtn: true,
+        layerBtn: true,
+        pipelineBtn: true,
+        sidebar: true,        
+      }
+      theme: "dark",
+      tag: {
+        enabled: {companies: true}
+      },
+      nodeType: {
+        disabled: {parameters: true}
+      }
+    }
+ /> 
+```
+
+| Name         | Type    | Default | Description |
+| ------------ | ------- | ------- | ----------- |
+| `data` | `{ edges: array (required), layers: array, nodes: array (required), tags: array }` | - | Pipeline data will be displayed on the chart |
+| options.theme | string | dark | select Kedro-Viz theme dark/light |
+| options.tag | `{enabled: {<tagName>: boolean}}` | - | Configuration for tag options |
+| options.nodeType | `{disabled: {parameters: boolean,task: boolean,data: boolean}}` | `{disabled: {parameters: true,task: false,data: false}}` | Configuration for node type options |
+| options.display |  |  |  |
+| `globalToolbar` | boolean | true | If `false` Global sidebar on extreme left will be hidden |
```

Comment:
nit - for description can we keep all of them simple. Like 

Show/Hide Global Toolbar 
Show/Hide Sidebar 
and for visbile.sidebar it would be Expand/Collapse Sidebar... 

something like it. 




 
 
--



--